.github/workflows/pretest.yml

```diff
@@ -77,6 +77,7 @@ jobs:
 
     - name: Build
       run: |
+        pip install git+https://github.com/ROCmSoftwarePlatform/hipify_torch.git
```

Comment:
Do we need hipify_torch even on CUDA build?

Reply:
No, we do not need hipify_torch on CUDA build. I updated the pretest.yml to reflect these changes. Thanks!